### DISCUSSION TITLE
Processing history of the individual projects in the project overview

### DISCUSSION THREAD

**Discussion by jens-2305** [kimai/kimai#2973] [Questions & Answers]
Hello all,

We would like to see in the project list when a project was last edited. Is it possible to introduce a date column in the project overview? It would also be helpful to be able to see the editing history in the individual project. Is this possible?

Many thanks

**Comment by kevinpapst** [comment]
The [Audit trail bundle](https://www.kimai.org/store/audit-trail-bundle.html) allows to record and visualize the change history of projects (and many other items).

**Comment by jens-2305** [comment]
Thanks for your reply this is a good idea
